configs/ci/permissions/ros.json

```diff
@@ -0,0 +1,12 @@
+{
+    "*": [
+        {
+            "verb": "*"
+        }
+    ],
+    "system": [
```

Comment:
Just to confirm, you'd want to add a granular permission which would be available during custom role creation, for: `ros:system:read`?

Reply:
We want to add read permission to all the users throughout. Do we need a custom role in this case?

Reply:
The permissions files are just defining your granular permissions which can be used in the UI during custom role creation. So in this case `ros:system:read` would be the only custom role that could be created in the UI. If you wanted `ros:*:read` as well, you'd need to add `"verb": "read"` before/after line 4.